core/storage/suggestion/gae_models.py

```diff
@@ -1990,3 +1997,1360 @@ def export_data(
+        if sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'IncreasingLastActivity']):
+            sort = -cls.last_contribution_date
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'DecreasingLastActivity']):
+            sort = cls.last_contribution_date
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'IncreasingPerformance']):
+            sort = cls.recent_performance
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'DecreasingAccuracy']):
+            sort = -cls.overall_accuracy
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'IncreasingAccuracy']):
+            sort = cls.overall_accuracy
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'DecreasingSubmissions']):
+            sort = -cls.submitted_translations_count
+        elif sort_by == (
+                constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                    'IncreasingSubmissions']):
+            sort = cls.submitted_translations_count
+
+        # The first sort property must be the same as the property to which
+        # an inequality filter is applied. Thus, the inequality filter on
+        # last_activity can not be used here.
+        if topic_ids is not None:
+            sort_query = cls.query(
+                datastore_services.all_of(
+                    cls.language_code == language_code,
+                    cls.topic_ids_with_translation_submissions.IN(topic_ids)
+                    )).order(sort)
+        else:
+            sort_query = cls.query(
+                datastore_services.all_of(
+                    cls.language_code == language_code
+                    )).order(sort)
+
+        sorted_results: List[
+            TranslationSubmitterTotalContributionStatsModel] = []
+        num_models_per_fetch = 100
+        today = datetime.date.today()
+
+        if last_activity is not None:
+            last_date = today - datetime.timedelta(days=last_activity)
+            next_offset = offset
+            while len(sorted_results) < page_size:
+                result_models: Sequence[
+                    TranslationSubmitterTotalContributionStatsModel] = (
+                    sort_query.fetch(
+                        num_models_per_fetch, offset=next_offset))
+                if not result_models:
+                    next_offset += 1
+                    break
+                for result_model in result_models:
+                    next_offset += 1
+                    if result_model.last_contribution_date >= last_date:
+                        sorted_results.append(result_model)
+                        if len(sorted_results) == page_size:
+                            break
+        else:
+            sorted_results = list((
```

Comment:
You can drop one of the parens at the end (and the corresponding close paren).

Reply:
Done.